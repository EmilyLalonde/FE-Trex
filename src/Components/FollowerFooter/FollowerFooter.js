import React, { Component } from "react";
import { StyleSheet, Text, View, TouchableOpacity } from "react-native";

export default class Footer extends Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <View style={styles.footer}>
        <TouchableOpacity
          onPress={() => this.props.navigate("FollowerDashboard")}
        >
          <Text style={styles.footerText}>Home</Text>
        </TouchableOpacity>
        <TouchableOpacity
          onPress={() => this.props.navigate("DefaultFollowerMessages")}
        >
          <Text style={styles.footerText}>Messages</Text>
        </TouchableOpacity>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  footer: {
    backgroundColor: "#84183B",
    flexDirection: "row",
    padding: 30,
    justifyContent: "space-around"
  },
  footerText: {
    color: "white",
    marginBottom: 20,
    fontSize: 20
  }
});
