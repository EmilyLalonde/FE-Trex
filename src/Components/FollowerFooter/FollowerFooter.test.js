import React from "react";
import "react-native";
import FollowerFooter from "./FollowerFooter";
import { shallow } from "enzyme";

describe("FollowerFooter", () => {
  it("should return true", () => {
    expect(true).toEqual(true);
  });
  it("should match the snapshot when loading", () => {
    let wrapper = shallow(<FollowerFooter navigation={{ navigate: jest.fn() }} />);

    expect(wrapper).toMatchSnapshot();
  });
});
