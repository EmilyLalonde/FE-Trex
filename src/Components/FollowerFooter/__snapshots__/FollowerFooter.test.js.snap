// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FollowerFooter should match the snapshot when loading 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#84183B",
      "flexDirection": "row",
      "height": 60,
      "justifyContent": "space-around",
      "padding": 10,
    }
  }
>
  <TouchableOpacity
    activeOpacity={0.2}
    onPress={[Function]}
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 20,
        }
      }
    >
      Home
    </Text>
  </TouchableOpacity>
</View>
`;
